doc/users-guide/base/typeAliases.rst

```diff
@@ -0,0 +1,67 @@
+:title: Users Guide: Type Aliases
+
+.. _ug-typeAliases:
+
+Type Aliases
+============
+
+In addition to :ref:`declaring variables <ug-vars>` using ``var`` and
+:ref:`constants <ug-constParam>` using ``const`` and ``param``, Chapel
+supports the declaration of *type aliases* using the ``type`` keyword.
+As the term suggests, type aliases create symbolic names for types.
```

Comment:
I think it would be good to go further to clarify  "Is it another name for the same type, or a new type?" - e.g. by adding "The type alias only creates a new name for an existing type. It does not create a new type".

Additionally, it's not exactly the job of this PR, but the language spec currently describes that as an *open issue*. I think it needs to be another name for an existing type (rather than creating a new type) but I'm not sure everyone will think the issue is settled without discussion. But, I'd like to see that settled in the spec, too.

Also see:
 * PR #5057, specifically the "Future Work" section of the PR message
 * PR #3406, which enables type aliases for generic types